frontend/src/features/projects/list/ProjectListView.tsx

```diff
@@ -222,6 +222,7 @@ const ProjectListView: React.FC<IProps> = ({ filterable, title, mode }) => {
       project.status = projectStatuses.find((x: any) => x.name === project.status)!;
       await service.deleteProject(project);
       setData(data?.filter(p => p.projectNumber !== project.projectNumber));
+      setDeleteId(undefined);
```

Comment:
Allow for multiple deletions if necessary

Reply:
why?

Reply:
As in they delete one project and goto delete another one after, previously it would have kept the old id in the state variable and not reset